### PR TITLE
Update fr.xml

### PR DESCRIPTION
**Pull request by Spidersouris**
well, it seems like I can't add another commit to my old pull request, sorry!

### CODE REVIEW

File: LEContextMenuHandler/Lang/fr.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 
 <Strings>
-  <RunDefault>Executer avec le profil de cette application</RunDefault>
-  <Submenu>Locale Emulator</Submenu>
+  <RunDefault>Éxecuter avec le profil de cette application</RunDefault>
```
Comment (KagamiChan) [inline LEContextMenuHandler/Lang/fr.xml]: Should it be Exécuter?

**Comment by KagamiChan** [comment]
Locale Emulator is the name of the application, it is localized in some lang files but not in others. Should it be unified? @regomne

File: LEContextMenuHandler/Lang/fr.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  <Submenu>Émulateur local</Submenu>
```
Comment (KagamiChan) [inline LEContextMenuHandler/Lang/fr.xml]: Locale Emulator is the name of the application, it is localized in some lang files but not in others. I think it is better unify this translation string. @regomne

File: LEContextMenuHandler/Lang/fr.xml (hunk 1 above)
Comment (Spidersouris) [inline LEContextMenuHandler/Lang/fr.xml]: Oh sorry, yeah. Missed up with the accents haha. Gonna change it now.

**Pull request merged**